app/views/spaces/home.html.slim

```diff
@@ -0,0 +1,88 @@
+coffee:
+	root = exports ? this
+	root.signinCallback = (authResult) ->
+		($ "#logout").fadeIn(1)
```

Comment:
if this is global, put it in a global controller, otherwise place it in a view controller for this screen